packages/docs/src/styles/_sections.scss

```diff
@@ -263,11 +263,15 @@
 .docs-overlay-example-transition {
   $overlay-example-width: $pt-grid-size * 40;
 
-  position: absolute;
+  position: fixed;
   top: $overlay-example-width / 2;
   left: calc(50% - #{$overlay-example-width / 2});
   z-index: $pt-z-index-overlay + 1;
   width: $overlay-example-width;
+
+  .pt-overlay-inline & {
```

Comment:
Yeah this is necessary. Line 105 applies this style to `.pt-overlay-backdrop`; this `.doc-overlay-example-transition` container is a sibling of `.pt-overlay-backdrop`, and it needs to be styled separately.

This does bring up a broader annoyance that users will need to explicit;y style overlay content with `position: absolute` if they want it to render properly inline. Maybe we should add a class (e.g. `pt-overlay-content`) on all siblings of `.pt-overlay-backdrop` that handles this position styling automatically?